spec/Makefile

```diff
@@ -7,5 +7,6 @@ webgpu.idl: index.bs extract-idl.py
 	python extract-idl.py index.bs > webgpu.idl
 
 online:
+	curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F output=err
 	curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 > index.html
```

Comment:
What does doing this twice get us? HTML output even when there is an error?

Reply:
Hm, wait, I see. Since these are curls there's no stderr so we have to get errors and results separately.